packages/cli/generators/extension/index.js

```diff
@@ -43,6 +43,13 @@ module.exports = class ExtensionGenerator extends ProjectGenerator {
 
   promptComponent() {
     if (this.shouldExit()) return;
+
+    if (this.options.componentName) {
```

Comment:
There should be a return statement to stop the prompt in case the component already exists.